src/geo/ui/legends/layer-legends-view.js

```diff
@@ -14,26 +15,40 @@ var LayerLegendsView = Backbone.View.extend({
     this._legendViews = [];
 
     this.settingsModel = options.settingsModel;
+    this.tryContainerVisibility = options.tryContainerVisibility;
```

Comment:
There is a general container for all layers. Then every layer has its own container for its legends. This general container is managed by a higher order view, so from the inner views we need to indicate a way to hide the general container.